packages/server/lib/modes/run.js

```diff
@@ -957,6 +957,12 @@ module.exports = {
     return openProject.launch(browser, spec, browserOpts)
   },
 
+  navigateToNextSpec (options = {}) {
```

Comment:
```js
return openProject.changeUrlToSpec(options.spec)
```

Personal preference but this destructure seems to add 0 value

Reply:
Agreed